draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1828,6 +1843,168 @@ Section Header
+            <xref target="section_block"/>.</t>
+
+            <t>Options: optionally, a list of options (formatted according to
+            the rules defined in <xref target="section_opt"/>) can be present.</t>
+
+          </list>
+        </t>
+
+        <t>All the statistic fields are defined as options in order to deal
+        with systems that do not have a complete set of statistics. Therefore,
+        In addition to the options defined in <xref target="section_opt"/>,
+        the following options are valid within this block:</t>
+
+        <texttable anchor="options_cib" title="Capture Information Block Options">
+          <ttcol align="left">Name</ttcol>
+          <ttcol align="left">Code</ttcol>
+          <ttcol align="left">Length</ttcol>
+          <ttcol align="left">Multiple allowed?</ttcol>
+
+	        <c>interface_id</c>
+          <c>1</c>
+          <c>4</c>
+          <c>no</c>
+
+          <c>global_location</c>
+          <c>2</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location_error</c>
+          <c>3</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation</c>
+          <c>4</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation_error</c>
+          <c>5</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity</c>
+          <c>6</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity_error</c>
+          <c>7</c>
+          <c>12</c>
+          <c>no</c>
+        </texttable>
+
+        <t>
+          <list hangIndent="8" style="hanging">
+
+            <t hangText="cib_interface_id:"><vspace blankLines="0"/>The
+	          interface_id option specifies the interface these statistics refers
+            to; the correct interface will be the one whose Interface
+            Description Block (within the current Section of the file) is
+            identified by same number (see <xref target="section_idb"/>) of
+            this field.</t>
+
+	          <t hangText="cib_global_location"><vspace blankLines="0"/>The 
+            cib_global_location option specifies the location of the packet 
+            capture or interface; location is stored as three 32-bit floats 
+            representing latitude and longitude in degrees, and altitude in 
+            metres.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_global_location_error"><vspace blankLines="0"/>The
+            global_location_error option specifies the error in location capture; 
+            this is stored as three 32-bit floats in the same order and 
+            representation as cib_location.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_orientation"><vspace blankLines="0"/>The 
+            cib_orientation option specifies the location of the packet 
+            capture or interface; location is stored as three 32-bit floats 
+            representing yaw, pitch, and roll in radians.</t>
```

Comment:
Hmm with lat/lng in degrees it probably makes more sense for this to be degrees with yaw corresponding to compass angle?